secrets/onepassword/backend.go

```diff
@@ -33,15 +32,13 @@ func (b *Backend) Init(parameters map[string]string) error {
 	if err != nil {
 		return fmt.Errorf("Error reading 1password backend parameters: %v", err)
 	}
-
 	b.Vault = parameters["vault"]
 
 	err = b.Client.SignIn(parameters["domain"], parameters["email"], parameters["secretKey"], parameters["masterPassword"])
 	if err != nil {
-		return err
+		fmt.Println(fmt.Sprintf("could not sign in to 1password %s", err))
```

Comment:
do we need `fmt.Println` or we can just return `fmt.Errorf("could not sigin into 1password: %v", err)`?

Reply:
Should be an `errors.Wrap`

Reply:
Fixed in 295720f